src/sentry/static/sentry/app/components/events/interfaces/spans/context.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {LocationDescriptor} from 'history';
+
+export type SpanEntryContextChildrenProps = {
+  getViewChildTransactionTarget: (props: {
+    'project.name': string;
+    transaction: string;
+    id: string;
+    eventSlug: string;
```

Comment:
Should this type have a name?

Reply:
I don't follow.

Reply:
Sorry, the parameter object to this function could have a named type.